src/io/flutter/view/TimeDilationAction.java

```diff
@@ -19,6 +19,7 @@
     super(app, "Enable Slow Animations", null, showIcon ? AllIcons.Vcs.History : null);
 
     setExtensionCommand("ext.flutter.timeDilation");
+    setEnabledStateValue(5.0);
```

Comment:
if you specified that the disabledStateValue was 1.0 then you could eliminate the override of the perform method.

Reply:
I overrode the perform method because timeDilation doesn't call a boolean service extension like FlutterViewToggleableAction's perform method, so we need to specify the params and call the service extension here.